### ISSUE TITLE
Is download of separate icons in svg/png format needed?

### ISSUE THREAD

**Issue by puzrin** [fontello/fontello#158]
I'd like to clarify my position:
- nobody going to compete photoshop
- if you need just icons, there are tons or site, where you can find one, b/w and color.
- every feature must add significant "value"
- there are some difference in licences between fonts and icons, that require additional talk to authors.
- compatibility needs with IE6-7 is quite small, and became even smaller in nearest years.

In other words - there is a big difference between "wow" features and "useful" features. My experiens is limited only to my projects. I don't need separate SVGs. But, may be you have reasons, why icons download can be useful.
